### ISSUE TITLE
Hall Effect Magnet Issues - Hard to press fit and magnet cracks print

### ISSUE THREAD

**Issue by TheFuzzyGiggler** [VoronDesign/Voron-Tap#58]
I'm on my 2nd printing of the "Tap Center" and have ran into the same issue twice.

I use a hall effect endstop and thus use the magnet on the side of the Tap Center.

Both times the following issues have came up.

1.) There is no good place to "push against" to press fit the magnet. Especially if the screws that secure the front magnets are in place (as is described in the manual in order of assembly). Since the magnet is offset a bit from the main body plane the securing screws engage first if using a vise to press fit.

Which leads to... 

2.) Both times once I finally find an angle to press the magnet in, it cracks the print horizontally along the midpoint of the hexagon where the magnet goes. It might be because of issue 1 and the fiddly angle at which I'm trying to press. I've press fit easily 100 6mmx3mm magnets in the past week (working on a magnetic panel latch design) and none of those cracked so I don't think it's my technique or force.

To sum it up, I think the magnet location needs to be tweaked, but I don't have any recommendations on how.
Except maybe have it as a seperate holder like the front magnets.

**Comment by djessup** [comment]
@TheFuzzyGiggler I'm not saying the model couldn't be improved, but some tips that could help you out in the meantime -
1. Check your extrusion is dialed in, as over-extruding is going to make an already-tight fit even tighter. Try under-extruding a few % even if you have it dialed in for test prints.
2. A very small drop of acetone can also help fit tight magnets in by softening some of the plastic around the hole. One minor caveat to this method is that you may need to apply a drop of superglue (once the acetone has _completely_ dried) as the softening from the acetone removes the natural clamping from displacing the plastic.

**Comment by jdspugh** [comment]
I found it a tight fit and had to use significant force. Mine didn't crack though, and it's staying very securely in there.
